src/components/JobListing/components/Tags.tsx

```diff
@@ -3,24 +3,29 @@ import React from 'react';
 interface Props {
   tags: string[];
   addFilter: (term: string) => void;
+  removeFilter: (term: string) => void;
   filters: string[];
 }
 
-export function Tags({ tags, addFilter,filters }: Props) {
+export function Tags({ tags, addFilter, removeFilter, filters }: Props) {
   const tagMarkup = tags.map((tag, index) => {
-  const isActiveTag = filters.includes(tag)
+    const isActiveTag = filters.includes(tag);
 
-  return (
-    <div
-      key={tag}
-      className={`flex-center tag-color leading-none py-2 pb-1 px-3 rounded-t-md rounded-b-md font-bold
-      ${isActiveTag ? 'active-tag-color' : 'hover:bg-desaturated-dark-cyan hover:text-white hover:cursor-pointer'}`}
-      onClick={() => addFilter(tag)}
-    >
-      {tag}
-    </div>
-  )
-});
+    const toggleFilter = (tag: string) => {
+      isActiveTag ? removeFilter(tag) : addFilter(tag);
+    };
```

Comment:
Would it be possible to combine `addFilter` and `removeFilter` into a singular `toggleFilter` function that switches between true and false for adding/removing tags?

Reply:
good call....I also had thoughts on refactoring some of the code as both the add/remove filters contain logic that could be simplified if we added a new tags key to the data and aggregate all those values, which are coming from 4 keys in total...